src/Simulation/Simulators.Tests/GlobalUsings.cs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+global using System;
+global using Xunit;
+global using Xunit.Abstractions;
```

Comment:
(I don't know C# and can be wrong)
I have an impression that this new file is never used.
In this PR there are 2 files with such a name. Both are never used?

Reply:
This is a new C# 10 feature, but allows breaking common `using` statements into a separate file that applies to all C# files in the same project. In particular, this is useful for things like `System` or `System.Linq` that are used in every or almost every file, and can help cut down on boilerplate in new C# files.

As it's a compile-time feature, using this new capability does not change anything at runtime, and in particular does not introduce new dependencies for users.